### PR TITLE
Update badge pricing

### PR DESCRIPTION
**Pull request by jcooter**
This pull request changes the config values for pricing for MAGLabs

Fixes https://github.com/magfest/magclassic/issues/22.

### CODE REVIEW

File: event-labs.yaml [hunk 1]
```diff
@@ -86,17 +86,18 @@ uber::config::badge_types:
...
 uber::config::max_dealers: 8
 uber::config::badge_prices:
   single_day:
-    - { 'Friday': 35 }
-    - { 'Saturday': 35 }
-    - { 'Sunday': 25 }
+    - { 'Friday': 30 }
+    - { 'Saturday': 30 }
+    - { 'Sunday': 20 }
   attendee:
```
Comment (kitsuta) [inline event-labs.yaml]: Hm, are we really not having any price bumps, even for at-door? If not, then you need to remove this line or puppet will cry.

Comment (ShevaDas) [inline event-labs.yaml]: Updated

Comment (kitsuta) [inline event-labs.yaml]: Awesome, +1 everything looks in order.

**Comment by kitsuta** [comment]
GitHub's merge resolve instructions didn't work, probably because this is from a fork. It ended up being easier just to make a new commit that implemented the changes.

**Pull request closed without merging**